### ISSUE TITLE
BinaryCrossEntropy instead of CrossEntropy for binary classification with ConceptWhitening

### ISSUE THREAD

**Issue by irmavdbrandt** [zhiCHEN96/ConceptWhitening#6]
Hi,

First of all, nice method and thank you for publishing the source code!

I noticed that you are performing the ISIC classification task, which is a binary one (malignant or not), with 2 output nodes and following cross-entropy loss instead of 1 output node and binary cross-entropy loss. Do you have a reason for choice or is it just convenience? 

I am asking this since I would like to use the CW method in a binary classification problem with one output node. 

Thank you!

Best, 
Irma

**Comment by zhiCHEN96** [comment]
Hi Irma,

It's just convenience, using 1 output node and BCE loss should be fine.
